cli/cli-utils/test/getConfig.test.ts

```diff
@@ -25,22 +25,5 @@ test('console a warning when the .npmrc has an env variable that does not exist'
   })
 
   // eslint-disable-next-line no-template-curly-in-string
-  expect(console.log).toHaveBeenCalledWith(expect.stringContaining('Failed to replace env in config: ${ENV_VAR_123}'))
-})
-
-test('should not console a warning when --json is specified', async () => {
```

Comment:
why was a test removed?

Reply:
Warnings are now always printed, but to `stderr` (which is covered by the first test). So this test is not needed anymore.